lib/factory_girl/linter.rb

```diff
@@ -33,9 +45,9 @@ def calculate_invalid_factories
     end
 
     def error_message
-      lines = invalid_factories.map do |factory, exception|
-        "* #{factory.name} - #{exception.message} (#{exception.class.name})"
-      end
+      lines = invalid_factories.map do |factory, exceptions|
```

Comment:
Unused block argument - `factory`. If it's necessary, use `_` or `_factory` as an argument name to indicate that it won't be used.